V2_MIGRATION_GUIDE.md

```diff
@@ -60,164 +63,129 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
 - The `com.auth0.android.result.Delegation` class has been removed. This was used as the result of the request to the [/delegation](https://auth0.com/docs/api/authentication#delegation) Authentication API legacy endpoint, disabled as of June 2017.
 - The `com.auth0.android.authentication.request.DelegationRequest` class has been removed. This was used to represent the request to the legacy Authentication API [/delegation](https://auth0.com/docs/api/authentication#delegation) endpoint, disabled as of June 2017.
 - The `com.auth0.android.util.Telemetry` class has been renamed to `com.auth0.android.util.Auth0UserAgent`.
-- The `com.auth0.android.request.AuthorizableRequest` class has been removed. You can achieve the same result using the method in Request: `Request#addHeader("Authorization", "Bearer TOKEN_VALUE")`.
+- The `com.auth0.android.request.AuthorizableRequest` class has been removed. You can achieve the same result using the `addHeader("Authorization", "Bearer TOKEN_VALUE")`.
 - The `com.auth0.android.authentication.request.TokenRequest` class has been removed. The ability to set a Code Verifier, and any request headers and parameters has been moved to the `com.auth0.android.request.Request` interface.
 - The `com.auth0.android.authentication.request.DatabaseConnectionRequest` class has been removed. The ability to set any request headers and parameters has been moved to the `com.auth0.android.request.Request` interface.
 
-### Class changes
+### Constants removed
+
+- `ParameterBuilder.GRANT_TYPE_JWT` has been removed.
+- `ParameterBuilder.ID_TOKEN_KEY` has been removed.
+- `ParameterBuilder.DEVICE_KEY` has been removed.
+- `ParameterBuilder.ACCESS_TOKEN_KEY` has been removed.
+- `ResponseType.CODE`, `ResponseType.ID_TOKEN`, and `ResponseType.ACCESS_TOKEN` have been removed.
+
+### Class and Interface changes
 
 - `SignupRequest` now implements `AuthenticationRequest` instead of the now-removed `AuthRequest`
 - `AuthorizableRequest` now extends `Request` instead of the now-removed `ParameterizableRequest`
 - `BaseCallback` has been deprecated; use `Callback` instead.
 
-### Constructors removed or changed
+### Constructor changes
 
-#### AuthenticationAPIClient
+- `AuthenticationAPIClient` can no longer be constructed from a `Context`. Use Use `AuthenticationAPIClient(client: Auth0)` instead. You can create an instance of `Auth0` using a `Context`.
```

Comment:
```suggestion
- `AuthenticationAPIClient` can no longer be constructed from a `Context`. Use `AuthenticationAPIClient(auth0: Auth0)` instead. You can create an instance of `Auth0` using a `Context`.
```

I'm not sure why you used "client" but we should either use the param name (as in kotlin you can pass named params) or remove the param name and keep the class, just for reference. 

If you prefer and given that this is not a breaking change, we could rename these to "account" (the same name we use in WebAuthProvider builders)

Reply:
Updated to remove duplicated "Use" and use `auth0` as named parameter.